compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/QueryDesugar.java

```diff
@@ -1934,7 +1937,31 @@ public void visit(BLangLetExpression letExpr) {
 
     @Override
     public void visit(BLangListConstructorExpr listConstructorExpr) {
-        listConstructorExpr.exprs.forEach(this::acceptNode);
+        List<BLangExpression> expressions = listConstructorExpr.exprs;
```

Comment:
Do we need to create a new variable here?

Reply:
Since it is used in 2 place, I think it is better to create a new variable